src/status_im/ui/screens/events.cljs

```diff
@@ -309,6 +310,21 @@
                                (get-in default-networks [default-network :config]))]
       {:initialize-geth-fx network-config})))
 
+(handlers/register-handler-fx
+  :fetch-web3-node-version-callback
+  (fn [{:keys [db]} [_ resp]]
+    (let [git-commit (nth (re-find #"-([0-9a-f]{7,})/" resp) 1)]
+      (when git-commit
+        {:db (assoc db :web3-node-version git-commit)}))))
+
+(handlers/register-handler-fx
+  :fetch-web3-node-version
+  (fn [{{:keys [web3] :as db} :db} _]
+    ((.-version.getNode web3) (fn [err, resp]
```

Comment:
I would also change this to double dot notation
```clj
(.. web3
    -version
    (getNode (fn [err resp] ...)))
```
Also, `[err, resp]` we don't use comma to separate vector elements, fn args, etc